### DISCUSSION TITLE
gatsby-plugin-image: onStartLoad is pased to img element

### DISCUSSION THREAD

**Discussion by pindjur** [gatsbyjs/gatsby#31572] [Help]
I got error from react-devtools backend that onStartLoad is not valid prop for img element. Stack trace leads to MainImage component.

**Comment by LekoArts** [comment]
Where did you see that `gatsby-plugin-image` would support `onStartLoad`?

**Reply by pindjur** [reply]
here:
https://github.com/gatsbyjs/gatsby/blob/5ba1ac2aca063577a907560627f262649c732279/packages/gatsby-plugin-image/src/components/gatsby-image.browser.tsx#L45
